geode-for-redis/src/main/java/org/apache/geode/redis/internal/commands/RedisCommandType.java

```diff
@@ -242,6 +242,7 @@
   SADD(new SAddExecutor(), SUPPORTED, new Parameter().min(3).flags(WRITE, DENYOOM, FAST)),
   SDIFF(new SDiffExecutor(), SUPPORTED,
       new Parameter().min(2).lastKey(-1).flags(READONLY, SORT_FOR_SCRIPT)),
+  SISMEMBER(new SIsMemberExecutor(), SUPPORTED, new Parameter().exact(3).flags(READONLY, FAST)),
```

Comment:
I'd recommend writing a simple unit test for RedisCommandType. It would just confirm that SISMEMBER is supported and maybe has an SIsMemberExecutor and 3 parameters including the READONLY and FAST flags.

Reply:
The correctness of the RedisCommandType flags is tested and validated against native Redis in `CommandIntegrationTest`, and the parameters are tested in the `sismemberWrongNumberOfArguments_returnsError` test in `AbstractSIsMemberIntegrationTest`, so we have comprehensive coverage of this code.